diff --git a/src/music-players.h b/src/music-players.h
index 1732197..845fbd4 100644
--- a/src/music-players.h
+++ b/src/music-players.h
@@ -2416,6 +2433,14 @@
       DEVICE_FLAGS_ANDROID_BUGS },
 
   /*
+   * Jolla
+   */
+  { "Jolla", 0x2931, "Sailfish", 0x0a01,
+      DEVICE_FLAGS_ANDROID_BUGS },
+  { "Jolla", 0x2931, "Sailfish", 0x0a05,
+      DEVICE_FLAGS_ANDROID_BUGS },
+
+  /*
    * Other strange stuff.
    */
   { "Isabella", 0x0b20, "Her Prototype", 0xddee, DEVICE_FLAG_NONE }
