--- libmtp-1.1.6/src/mtpz.c.orig	2013-05-14 18:57:23.417319178 +0200
+++ libmtp-1.1.6/src/mtpz.c	2013-05-14 18:58:24.777318734 +0200
@@ -119,7 +119,7 @@
 	char *home = getenv("HOME");
 	if (!home)
 	{
-		LIBMTP_INFO("Unable to determine user's home directory, MTPZ disabled");
+		LIBMTP_ERROR("Unable to determine user's home directory, MTPZ disabled");
 		return -1;
 	}
 
@@ -130,7 +130,7 @@
 	FILE *fdata = fopen(path, "r");
 	if (!fdata)
 	{
-		LIBMTP_INFO("Unable to open ~/.mtpz-data for reading, MTPZ disabled.");
+		LIBMTP_ERROR("Unable to open ~/.mtpz-data for reading, MTPZ disabled.");
 		return -1;
 	}
 
@@ -138,7 +138,7 @@
 	MTPZ_PUBLIC_EXPONENT = (unsigned char *)fgets_strip((char *)malloc(8), 8, fdata);
 	if (!MTPZ_PUBLIC_EXPONENT)
 	{
-		LIBMTP_INFO("Unable to read MTPZ public exponent from ~/.mtpz-data, MTPZ disabled");
+		LIBMTP_ERROR("Unable to read MTPZ public exponent from ~/.mtpz-data, MTPZ disabled");
 		return -1;
 	}
 
@@ -146,20 +146,20 @@
 	char *hexenckey = (unsigned char *)fgets_strip((char *)malloc(35), 35, fdata);
 	if (!hexenckey)
 	{
-		LIBMTP_INFO("Unable to read MTPZ encryption key from ~/.mtpz-data, MTPZ disabled");
+		LIBMTP_ERROR("Unable to read MTPZ encryption key from ~/.mtpz-data, MTPZ disabled");
 		return -1;
 	}
 	MTPZ_ENCRYPTION_KEY = hex_to_bytes(hexenckey, strlen(hexenckey));
 	if (!MTPZ_ENCRYPTION_KEY)
 	{
-		LIBMTP_INFO("Unable to read MTPZ encryption key from ~/.mtpz-data, MTPZ disabled");
+		LIBMTP_ERROR("Unable to read MTPZ encryption key from ~/.mtpz-data, MTPZ disabled");
 	}
 
 	// Should only be 256 characters in length, but fgets will encounter a newline and stop.
 	MTPZ_MODULUS = (unsigned char *)fgets_strip((char *)malloc(260), 260, fdata);
 	if (!MTPZ_MODULUS)
 	{
-		LIBMTP_INFO("Unable to read MTPZ modulus from ~/.mtpz-data, MTPZ disabled");
+		LIBMTP_ERROR("Unable to read MTPZ modulus from ~/.mtpz-data, MTPZ disabled");
 		return -1;
 	}
 
@@ -167,7 +167,7 @@
 	MTPZ_PRIVATE_KEY = (unsigned char *)fgets_strip((char *)malloc(260), 260, fdata);
 	if (!MTPZ_PRIVATE_KEY)
 	{
-		LIBMTP_INFO("Unable to read MTPZ private key from ~/.mtpz-data, MTPZ disabled");
+		LIBMTP_ERROR("Unable to read MTPZ private key from ~/.mtpz-data, MTPZ disabled");
 		return -1;
 	}
 
@@ -175,13 +175,13 @@
 	char *hexcerts = fgets_strip((char *)malloc(1260), 1260, fdata);
 	if (!hexcerts)
 	{
-		LIBMTP_INFO("Unable to read MTPZ certificates from ~/.mtpz-data, MTPZ disabled");
+		LIBMTP_ERROR("Unable to read MTPZ certificates from ~/.mtpz-data, MTPZ disabled");
 		return -1;
 	}
 	MTPZ_CERTIFICATES = hex_to_bytes(hexcerts, strlen(hexcerts));
 	if (!MTPZ_CERTIFICATES)
 	{
-		LIBMTP_INFO("Unable to parse MTPZ certificates from ~/.mtpz-data, MTPZ disabled");
+		LIBMTP_ERROR("Unable to parse MTPZ certificates from ~/.mtpz-data, MTPZ disabled");
 		return -1;
 	}
 
